client/src/components/page/CommunityPage.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { COMMUNITY_PAGE_SERVLET } from 'constants/links.js';
+import UserItem from 'components/ui/UserItem.js';
+
+/** Renders the community page. */
+const buildUserListItem = function(user) {
```

Comment:
Write a comment for this function. Can be simple like:
/** Returns the html for a single user. */